enhancements/update/targeted-update-edge-blocking.md

```diff
@@ -0,0 +1,725 @@
+
+Ad-hoc testing when landing new features should be sufficient for `oc` and the web-console, although if they have existing frameworks for comparing output with mock cluster resources, that would be great too.
+
+### Graduation Criteria
+
+This will be released directly to GA.
+
+#### Dev Preview -> Tech Preview
+
+This will be released directly to GA.
+
+#### Tech Preview -> GA
+
+This will be released directly to GA.
+
+#### Removing a deprecated feature
+
+This enhancement does not remove or deprecate any features.
+
+### Upgrade / Downgrade Strategy
+
+The graph-data schema is already versioned.
+
+We have [an open RFE][ota-123] to version the Cincinnati API, but even without that, adding new optional properties (`conditionalEdges`) for new features (edges which would have previously been completely blocked) is a backwards-compatible change.
+
+### Version Skew Strategy
+
+Newer update services consuming older graph-data will know that they can use their 1.1.0 parser on 1.0.0 graph-data without needing to make changes.
+
+Older update services consuming newer graph-data will know that they are missing some features unique to 1.1.0, but that they will still get something reasonable out of the data by using their 1.0.0 parser (they'll just consider all conditional edges to be complete blockers).
+
+Newer clients talking to older update services will not receive any `conditionalEdges`, but they will understand all the data that the update service sends to them.
+Older clients talking to newer update services will not notice `conditionalEdges`, so those edges will continue to be unconditionally blocked for those clients.
+
+Newer clients consuming older ClusterVersion will not receive any `conditionalUpdates`, but they will understand all the data included in the ClusterVersion object (e.g. `availableUpdates`).
+Older clients consuming newer ClusterVersion will not notice `conditionalUpdaets`, so those edges will continue to be unconditionally blocked for those clients.
+
+## Implementation History
+
+Major milestones in the life cycle of a proposal should be tracked in `Implementation
+History.
+
+## Drawbacks
+
+Dynamic edge status that is dependent on cluster state makes [the graph-data repository][graph-data] a less authoritative view of the graph served to a given client at a given time, as discussed in [the *risks and mititgations* section](#risks-and-mitigations).
+This is mitigated by ClusterVersion's [`status.history[].overrides'](#enhanced-clusterversion-representation), which records any cluster-version operator objections which the cluster administrator chose to override.
+It is possible that cluster administrators would chose to clear that data, but it seems unlikely that they would invest significant effort in trying to cover their tracks when [the edges are supported regardless of whether they were recommended][openshift-docs-32091].
+
+## Alternatives
+
+### Additional data sources
```

Comment:
there are a mix of heading levels in this section that don't seem intentional

Reply:
Oops.  Fixed with e9933b8 -> 6f093a0 to use `###` for all *Alternatives* subsections.